pkgs/top-level/all-packages.nix

```diff
@@ -25513,7 +25513,7 @@ with pkgs;
 
   yubihsm-shell = callPackage ../tools/security/yubihsm-shell { };
 
-  yubioath-flutter = callPackage ../applications/misc/yubioath-flutter { };
+  yubico-authenticator = callPackage ../applications/misc/yubico-authenticator { };
```

Comment:
Unfortunately CI isn't smart enough to know that this isn't a new package, and as such now requires moving it to `pkgs/by-name`

Reply:
So I can just ignore that error?

Reply:
I'd prefer it be moved to `pkgs/by-name`, CI shouldn't be red

Reply:
Okay, will do once #291598 is merged.